src/fidesops/models/policy.py

```diff
@@ -52,7 +52,7 @@ class DrpAction(EnumType):
     access_specific = "access:specific"
 
 
-PseudonymizationPolicy = SupportedMaskingStrategies
+PseudonymizationPolicy = MaskingStrategyFactory.get_strategies()
```

Comment:
@PSalant726 one (hopefully) last thing - i wasn't really sure the purpose of this, should we get someone from the fidesops team just to confirm that what i'm doing here is OK? the deprecated comment below doesn't make things clearer for me.

Reply:
You're not changing any functionality, so it shouldn't break any DB migrations. Still, if you want to get a review from the fidesops team it couldn't hurt.

Reply:
thanks @pattisdr  - removed in https://github.com/ethyca/fidesops/pull/560/commits/e4b449bf09667d300bc252756cf5eec361752e76